### DISCUSSION TITLE
JavaScript Error

### DISCUSSION THREAD

**Discussion by dande74** [GingerXR/products#9] [General]
Cannot figure out why I get this error on one of my computer?

![JSE](https://github.com/GingerXR/products/assets/109685508/ff1634a1-1b74-49a4-91ef-1d1c1c6f2930)

**Comment by GingerXR** [comment]
Hi, 

Could you try downloading the the latest vc_redist:

https://aka.ms/vs/17/release/vc_redist.x64.exe
from this page: https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

and / or update your GPU driver.

Let us know if that fixes the issue

**Reply by dande74** [reply]
Yes, that fixed the issue, thanks!

Now I'm just waiting for the "Wired" feature :)
